### ISSUE TITLE
Update error message when attempting to upload an image

### ISSUE THREAD

**Issue by augustuswm** [oxidecomputer/console#1859]
Now with quotas in place the error message when attempting to upload an image needs to be updated:

![image](https://github.com/oxidecomputer/console/assets/1377730/983a84c3-3b3e-4bde-886e-269c887fd8ce)

**Comment by david-crespo** [comment]
This was a situation where creating the disk required to start import would go over quota?

**Comment by augustuswm** [comment]
Yeah, I had set an abnormally low quota, ~5kb and tried to import a 2GB image.
